### PR TITLE
FIX-2864: Add useAbiword flag

### PR DESCRIPTION
**Pull request by jainendra**
This is a fix for the issue - #2864
Which is import failure of PDF after importing a text file

### CODE REVIEW

**Comment by jainendra** [comment]
At present if anyone tries uploading a PDF and doc will work until someone uploads a text file.
There was a logic error, abiword object was being used as a flag. abiword variable gets assigned when ImportHandler.js is loaded now when a text file gets uploaded we change the abiword variable value to false so as to not use the abiword plugin. Now on second attempt when we try to upload a PDF or a Doc uploading fails as abiword was assigned to false.

File: src/node/handler/ImportHandler.js [hunk 1]
```diff
@@ -56,10 +56,15 @@ exports.doImport = function(req, res, padId)
...
     , text
     , importHandledByPlugin
-    , directDatabaseAccess;
+    , directDatabaseAccess
+    , useAbiword;
 
   var randNum = Math.floor(Math.random()*0xFFFFFFFF);
   
+  // setting flag for whether to use abiword or not
+  if(abiword)useAbiword = true;
```
Comment (Gared) [inline src/node/handler/ImportHandler.js]: Please use a shorter syntax like
```
useAbiword = abiword != null;
```

Comment (jainendra) [inline src/node/handler/ImportHandler.js]: Okay I'll update it

**Comment by Gared** [comment]
Thanks for your work! @jainendra

**Pull request merged**